test/ipynb/python/KernelMagicsTest.ipynb

```diff
@@ -76,6 +76,19 @@
     "p << new Line(x: [0, 1, 2, 3, 4, 5], y: [0, 1, 6, 5, 2, 8])"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%groovy\n",
+    "i = 0;\n",
+    "while(i < 1000) {\n",
```

Comment:
should be an infinite loop, `while(true) {sleep(100)}`

Reply:
done.
btw the interrupting doesn't work  #7355